internal/sysinfo/memtotal_darwin.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // PhysicalMemoryBytes returns the total amount of host memory.
 func PhysicalMemoryBytes() (uint64, error) {
-	mib := []int32{6 /* CTL_HW */, 24 /* HW_MEMSIZE */}
+	mib := []int32{6 /* CTL_HW */, 24 /* HW_MEMSIZE */,}
```

Comment:
The comma would only [be required][1] if there was a newline before the closing bracket.
What compilation error do you get when building this without the comma?

This is my output:
```
$ go version
go version go1.7.3 linux/amd64

$ GOOS=darwin GOARCH=arm64 go build -v ./internal/sysinfo/memtotal_darwin.go
runtime/internal/sys
runtime/internal/atomic
runtime
internal/race
errors
sync/atomic
sync
syscall
command-line-arguments

$ echo $?                                                                   
0
```

[1]: https://dave.cheney.net/2014/10/04/that-trailing-comma